src/redux/categories/categoriesSlice.js

```diff
@@ -1,3 +1,5 @@
+// src/redux/categories/categoriesSlice.js
+
 import { createSlice } from '@reduxjs/toolkit';
 
 const categoriesSlice = createSlice({
```

Comment:
Great  work was done so far 👍🏼  ,  but here the checkStatus reducer needs to receive (state, action) in the payload you specify how to update the state if you receive under construction return it otherwise return what's in the state 

`initialState: { categories: [], }, reducers: { checkStatus: (state, action) => { state.categories = action.payload === "Under construction" ? "Under construction" : state.categories }, }, `